JenkinsfileRT

```diff
@@ -76,7 +76,7 @@ bc0.conda_packages = [
 ]
 bc0.pip_reqs_files = ['requirements-sdp.txt']
 bc0.build_cmds = [
-    "pip install -e .[test,ephem] --no-cache-dir",
+    "pip install -e .[test,sdp] --no-cache-dir",
```

Comment:
Are the SDP folks in the habit of installing with this argument set? Or is it done for them automagically?

Reply:
Unless the discussion with SPB reveals otherwise, I believe this is the path. The theory is that this build is used to produce the dependency files found on artifactory with which SDP uses to build JWST. Initial test will be to run the regression job after merging this, if the regression is successful, or maybe even if it isn't.